File: 1512-number-of-good-pairs/1512-number-of-good-pairs.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function numIdenticalPairs($nums) {
        $count = 0;
        for($i = 0;$i < sizeof($nums) ;$i++)
            for($j = 0; $j <sizeof($nums) ;$j++)
                if($nums[$i]==$nums[$j] && $i < $j)
                    $count++;
        return $count;
    }
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        count = 0
        for i in range(len(nums)-1):
            for j in range(1,len(nums)):
                if i<j :
                    if nums[i] == nums[j] :
                        count=count+1
        return count